### DISCUSSION TITLE
link.xml in Hololens2

### DISCUSSION THREAD

**Discussion by hemasree99s** [microsoft/MixedRealityToolkit-Unity#10463] [Q&A] [answered]
While upgrading to MRTK 2.7.3, additionally ignoreifmissing attribute got added in link.xml of MRTK generated. Can this be removed

**Comment by keveleigh** [comment, marked as answer]
We added this to support Unity 2021, which became stricter about the syntax in the link.xml. If you're not using Unity 2021, you can likely remove it! I don't recall how aggressive our updater is to re-adding it if you remove it.
